src/lib/tour/Tour.svelte

```
@@ -0,0 +1,254 @@
+      
+      <p class={styles.description({ class: clsx(theme?.description, classes?.description) })}>{currentStepData.description}</p>
+      
+      {#if steps.length > 1}
+        <div class={styles.progressContainer({ class: clsx(theme?.progressContainer, classes?.progressContainer) })}>
+          {#each steps as _, index}
+            <button
+              class="{styles.progressDot({ class: clsx(theme?.progressDot, classes?.progressDot) })} {index === currentStep ? styles.progressDotActive() : ''}"
+              onclick={() => goToStep(index)}
+              aria-label="Go to step {index + 1}"
+            ></button>
+          {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)